tests/Buildalyzer.Workspaces.Tests/ProjectAnalyzerExtensionsFixture.cs

```diff
@@ -122,18 +139,21 @@ public void SupportsAnalyzers()
         private IProjectAnalyzer GetProjectAnalyzer(string projectFile, StringWriter log, AnalyzerManager manager = null)
         {
             // The path will get normalized inside the .GetProject() call below
-            string projectPath = Path.GetFullPath(
-                Path.Combine(
-                    Path.GetDirectoryName(typeof(ProjectAnalyzerExtensionsFixture).Assembly.Location),
-                    @"..\..\..\..\" + projectFile));
+            string projectPath = GetFullPath(projectFile);
             if (manager == null)
             {
-                manager = new AnalyzerManager(new AnalyzerManagerOptions
-                {
-                    LogWriter = log
-                });
+                manager = new AnalyzerManager(new AnalyzerManagerOptions { LogWriter = log });
             }
+
             return manager.GetProject(projectPath);
         }
+
+        private static string GetFullPath(string partialPath)
+        {
+            return Path.GetFullPath(
+                Path.Combine(
+                    Path.GetDirectoryName(typeof(ProjectAnalyzerExtensionsFixture).Assembly.Location),
+                    @"..\..\..\..\" + partialPath));

```

Comment:
Consider using `Directory.GetParent` instead, slashes are different in different operating systems.

Reply:
I didn't want to change too much, that was existing code refactored into a shared method so I assume it's been working.

Reply:
@markrendle thanks, keeping the changes limited makes it easier for me to pull this change in without spending a lot of time I don't have right now reviewing it.

@psfinaki I think you've got a very good point though and at some point soon I'll need to do a little auditing of the code to remove slashes wherever I can. That said, I do think the `Path` family of APIs normalizes slash characters fairly well across platforms which is why this code worked well enough before.

In any case, good enough to merge and release for now :)